webpack.config.js

```diff
@@ -8,6 +8,7 @@ var plugins = [
     'caching': true
   }),
   new webpack.optimize.OccurrenceOrderPlugin(),
+  new webpack.IgnorePlugin(/vertx/),
```

Comment:
It was time for that. Still unbelievable for me that they inject a environment related dependency into that polyfill ... nonono not good... at least we got it fixed now.